Assets/MXR.SDK/Runtime/Types/WifiTypes.cs

```diff
@@ -313,9 +313,13 @@ public class EnterpriseWifiConnectionRequest
         public string ssid;
         public string password;
         public string identity;
+        [DefaultValue("")]
```

Comment:
Shouldn't this be set to `[DefaultValue(false)]` instead? This way it gets serialized only when it's set to true

Reply:
Addressed